application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarTemplateProvider.xml

```diff
@@ -177,17 +177,29 @@
         <classType>com.xpn.xwiki.objects.classes.DBListClass</classType>
       </spaces>
       <template>
+        <cache>0</cache>
+        <classname/>
         <customDisplay/>
         <disabled>0</disabled>
+        <displayType>input</displayType>
+        <hint/>
```

Comment:
Be careful with this because [you might not be able to install this page](http://jira.xwiki.org/browse/XWIKI-15028?focusedCommentId=97506&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-97506) on older versions of XWiki.

If I understand correctly, only the change to the ``spaces`` property below is really needed.

Reply:
Indeed, it's not working. I'm gonna keep only the change on spaces property, not the xar:format one.